azkaban/remote.py

```diff
@@ -369,12 +371,15 @@ def delete_project(self, name):
     return res
 
   def run_workflow(self, name, flow, jobs=None, concurrent=True,
-    properties=None, on_failure='finish', notify_early=False, emails=None):
+                   properties=None, on_failure='finish', notify_early=False,
+                   emails=None, disabled_jobs=None):
     """Launch a workflow.
 
     :param name: Name of the project.
     :param flow: Name of the workflow.
     :param jobs: List of names of jobs to run (run entire workflow by default).
+    :param disabled_jobs: Inverse of jobs. List of names of jobs not to run.
+      Will not be used if jobs is also provided as input.
```

Comment:
Rather than have one override the other (not obvious without reading the documentation), can we throw a `ValueError` when both are specified?